Automattic/wp-calypso
# Checkout: Move WeChat QR code component into new checkout

#### Changes proposed in this Pull Request

This is a small refactor that moves the `WeChatPaymentQRcode` component from old checkout (`client/my-sites/checkout/checkout`) into new checkout (`client/my-sites/checkout/composite-checkout`). This also removes its previous usage by removing the `WechatPaymentBox` component from old checkout. This is safe because old checkout is not used anywhere and will soon be removed itself. In fact, the reason for this change is to begin moving new checkout's dependencies out of old checkout to make the removal easier.

#### Testing instructions

- Apply D45650-code to force WeChat Pay to be available.
- Sandbox the store.
- Add a plan to your cart and visit checkout. 
- Verify that "WeChat Pay" is an option displayed in the final payment step.
- Verify that you cannot pay if the "Name" field is empty.
- Fill out the "Name" field and submit the payment.
- Verify that you see a QR code appear and it is scrolled into view.
- In a *new window or tab* open the link at the bottom that reads "To open and pay with the wechat pay app directly, click here".
- In the new window, you should see a Stripe test page.
- Click to accept the payment and verify that the payment completes successfully in the original window and that the plan is purchased.
